mock/py/mockbuild/package_manager.py

```diff
@@ -338,3 +350,50 @@ def builddep(self, *pkgs, **kwargs):
                 if 'no such command: builddep' in line.lower():
                     raise BuildError("builddep command missing.\nPlease install package dnf-plugins-core.")
             raise
+
+
+# class for microdnf
+class MicroDnf(_PackageManager):
+    name = 'microdnf'
+
+    def __init__(self, config, buildroot, plugins, bootstrap_buildroot):
+        super(MicroDnf, self).__init__(config, buildroot, plugins, bootstrap_buildroot)
+        self.command = config['microdnf_command']
+        self.install_command = config['microdnf_install_command']
+        self.builddep_command = [self.command, 'builddep']
```

Comment:
I believe that this should be like:
`self.builddep_command =  config['dnf_builddep_command']`
which is better than the other option
self.builddep_command = config['dnf_command'], 'builddep']